ppml/trusted-big-data-ml/scala/docker-occlum/run_spark_on_occlum_glibc.sh

```diff
@@ -69,6 +69,9 @@ build_spark() {
     # Prepare BigDL
     mkdir -p image/bin/jars
     cp -f $BIGDL_HOME/jars/* image/bin/jars
+
+    # Prepare tpc=h
+    cp $TPCH_DIR/target/scala-2.12/spark-tpc-h-queries_2.12-1.0.jar image/bin/jars
```

Comment:
I think we can copy TPC-H jar into `$BIGDL_HOME/jars/` during docker build. Then, we can remove this line